extide/gradle/src/org/netbeans/modules/gradle/api/execute/GradleCommandLine.java

```diff
@@ -186,26 +209,57 @@ private Property(Argument.Kind kind, String prefix, String flag) {
 
     }
 
-    public enum Parameter {
+    public enum Parameter implements GradleOptionItem {
 
-        SETTINGS_FILE(UNSUPPORTED, "-c", "--settings-file"),
-        CONSOLE(UNSUPPORTED, "--console"),
+        BUILD_FILE(UNSUPPORTED, "-b", "--build-file"),
+        CONFIGURATION_CACHE_PROBLEMS(PARAM, argValues("fail", "warn"), "--configuration-cache-problems"),
+        CONSOLE(UNSUPPORTED, argValues("plain", "auto", "rich", "verbose"), "--console"),
+        DEPENDENCY_VERIFICATION(PARAM, argValues("strict", "lenient", "off"), "-F", "--dependency-verification"),
+        EXCLUDE_TASK(PARAM, "-x", "--exclude-task"),
```

Comment:
`EXCLUDE_TASK` `PROJECT_DIR` and `INIT_SCRIPT` are missing description. Maybe write a test :) that will just fetch descrs from all options.
The `unsupported` parameters are never presented as items -- is their description never read ?